src/main/webapp/dev/js/main/instructorFeedbacks.js

```diff
@@ -68,12 +70,12 @@ function formatDigit(num) {
 }
 
 /**
- * Format a date object into DD/MM/YYYY format
+ * Format a date object into ddd, DD MMM, YYYY format. (Sat, 05 May, 2012)
  * @param date
  * @returns {String}
  */
-function convertDateToDDMMYYYY(date) {
-    return `${formatDigit(date.getDate())}/${formatDigit(date.getMonth() + 1)}/${date.getFullYear()}`;
+function convertDateTodddDDMMMYYYY(date) {
```

Comment:
I realised this function and the one below it are actually completely unnecessary; see my suggestion below.